docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
 Contact settings
 ****************
 
 Contact merge settings
 ======================
 
+.. image:: images/contact-merge-settings.png
+  :width: 600
+  :alt: Screenshot showing Contact Merge Settings Configuration in Mautic
+
+* **Merge by unique fields with operator** - If you use more than one unique fields identifiers then you can define the way of the find to merge it. 
+
 Contact list settings
 =====================
 
+.. image:: images/contact-list-settings.png
+  :width: 600
+  :alt: Screenshot showing Contact List Settings Configuration in Mautic
+
+* **Columns** -
+
 Import settings
 ===============
 
+.. image:: images/import-settings.png
+  :width: 600
+  :alt: Screenshot showing Import Settings Configuration in Mautic
+
+* **Automatically import in the background if the CSV has more rows than defined** - The browser/background import determines the number of rows in the CSV if this option is greater than 0.
+
 Segment settings
 ****************
 
+.. image:: images/segment-settings.png
+  :width: 600
+  :alt: Screenshot showing Segment Settings Configuration in Mautic
+
+* **Show warning if Segment hasn’t been rebuilt for X hours** - For all Contacts, dynamic Segments get rebuilt nightly. If there is an error that prevents a Segment from rebuilding, Mautic displays a warning message. This field allows you to configure the allowable length of time between rebuilds, post which the warning message appears.
+
 Company settings
 ****************
 
+.. image:: images/company-merge-settings.png
+  :width: 600
+  :alt: Screenshot showing Company Merge Settings Configuration in Mautic
+
+* **Merge by unique fields with operator** - When you use more than one unique field identifier, you can define how the search merges them.
+
 Notification settings
 *********************
 
-Campaign notification settings
-==============================
+.. image:: images/campaign-notification-settings.png
+  :width: 600
+  :alt: Screenshot showing Campaign Notification Settings Configuration in Mautic
+
+.. image:: images/webhook-notification-settings.png
+  :width: 600
+  :alt: Screenshot showing Webhook Notification Settings Configuration in Mautic
+
+If a Campaign or Webhook is automatically unpublished because of a high volume of errors, Mautic sends a notification alerting Users.
+
+* **Send notification to author** - Set this field to Yes to send an Email notification to the User who created the unpublished Campaign or Webhook. If the User is deleted, no notification is sent.
+
+* **Email addresses to receive notifications** - To send notifications to Users other than the author, enter a comma-separated list of Email addresses. You can include distribution lists and addresses of the Users who are not present in the Mautic instance.
```

Comment:
Where is this field? It is not demonstrated in the screenshot above. Does it apply to both webhooks and campaigns?